source/features/edit-comments-faster.tsx

```diff
@@ -1,21 +1,23 @@
 import React from 'dom-chef';
-import select from 'select-dom';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
 import PencilIcon from 'octicon/pencil.svg';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
 function init(): void {
-	const menuItems = select.all('details .js-comment-edit-button:not(.rgh-edit-comment)');
+	observe('.js-comment-update:not(.rgh-edit-comment)', {
+		add(comment) {
+			comment.classList.add('rgh-edit-comment');
 
-	for (const item of menuItems) {
-		item.classList.add('rgh-edit-comment');
-
-		const button = item.cloneNode();
-		button.append(<PencilIcon/>);
-		button.classList.replace('dropdown-item', 'timeline-comment-action');
-		item.closest('details')!.before(button);
-	}
+			comment.closest('.js-comment')?.querySelector('.js-comment-header-reaction-button')?.after(
```

Comment:
Why are you selecting js-comment-update and then navigating to this one? Just select this element in the observer.

Reply:
Does `this one` refer to `.js-comment-header-reaction-button`? I failed to find a selector meeting the following requirements:

- Comments in review (`.review-comment`)
- Made by others (not `.current-user`)
- Can be edited (the key issue, comment's classes didn't tell whether you can edit or not)

So I reverted how this PR was initially made ([`3e5fb20`](https://github.com/sindresorhus/refined-github/pull/3407/commits/3e5fb20d78b7965cd71db4c03fcf06a21626e159))

Reply:
> * Made by others (not `.current-user`)

Why?

> * Can be edited (the key issue, there is no difference in editable and not-editable review comments' classes)

It can be edited if:

- it's the current user, or
- the current user can edit the sidebar, for example.

> So I reverted how this PR was initially made ([`3e5fb20`](https://github.com/sindresorhus/refined-github/pull/3407/commits/3e5fb20d78b7965cd71db4c03fcf06a21626e159))

Let's not go reverting things, this can definitely be done, including selector-observer

Reply:
> * Comments in review (`.review-comment`)

That's completely irrelevant. Every comment can have this button.

Reply:
Those restrictions are not present in your current code except `can be edited`.

The code is fine except `?.` and the lack of explanation why the DOM traversal is needed. I fixed/added both.

Reply:
Though the PR has been merged, I'm answering these questions here:

> > * Comments in review (`.review-comment`)
> 
> That's completely irrelevant. Every type of comment can be edited.

For comments in the timeline (`.timeline-comment`) comments have `.editable-comment`, but this is not the case for comments in reviews.

> > * Made by others (not `.current-user`)
> 
> Why?

Comments made by users themselves have `.current-user`, but now I remember this doesn't always mean it can be edited.

> > * Can be edited (the key issue, there is no difference in editable and not-editable review comments' classes)
> 
> It can be edited if:
> 
> * it's the current user, or
> * the current user is a maintainer (you can determine this by checking whether the user can edit the sidebar, for example)

That's what this PR needs 👍 I limited my thoughts to the comments.